components/molecules/CardHorizontalBarChart/card-horizontal-bar-chart.tsx

```diff
@@ -35,6 +35,12 @@ const CardHorizontalBarChart = ({ languageList, withDescription }: CardHorizonta
     setDescriptText(descriptText);
   };
 
+  const handleKeyDown = (event: React.KeyboardEvent, languageName: string) => {
```

Comment:
@nickytonline @Firdous2307 
What will this `<span>` do? Remove the visible names and makes them available to screen readers only?

Reply:
All done @nickytonline

Reply:
> @nickytonline @Firdous2307 What will this `<span>` do? Remove the visible names and makes them available to screen readers only?

@CorinaMurg Yes, exactly!

Reply:
In most of the cases @CorinaMurg, they didn't have a label to begin with so this is already an improvement. Longer term it would be good to have the label visible so it's not just for screen readers, but I'd suggest a separate issue for that as it probably affects the current design a bit.